ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalFoundationConsent.cshtml.cs

```diff
@@ -9,30 +10,29 @@ namespace ApplyToBecomeInternal.Pages.TaskList.LegalRequirements
 {
 	public class LegalFoundationConsentModel : LegalModelBase
 	{
-		public LegalFoundationConsentModel(ILegalRequirementsRepository legalRequirementsRepository,
-			IAcademyConversionProjectRepository academyConversionProjectRepository) :
-			base(legalRequirementsRepository, academyConversionProjectRepository)
+		public LegalFoundationConsentModel(IAcademyConversionProjectRepository academyConversionProjectRepository) :
+			base(academyConversionProjectRepository)
 		{
 		}
 
 		[BindProperty] public string Approved { get; set; }
 
 		public void OnGet(int id)
 		{
-			Approved = LegalRequirements.FoundationConsent.ToString();
+			Approved = Requirements.FoundationConsent.ToString();
 		}
 
 		public async Task<IActionResult> OnPostAsync(int id)
 		{
-			LegalRequirements.FoundationConsent = Approved switch
+			Requirements.FoundationConsent = Approved switch
```

Comment:
I know you didn't make this change. But there is a ToDescription extension method that can be used instead of this switch statement. Would save some screen space.